documentation/modules/configuring/proc-generating-reassignment-json-files.adoc

```diff
@@ -58,7 +61,7 @@ spec:
 * You have a `KafkaUser` configured with ACL rules that specify permission to produce and consume topics from the Kafka brokers.
 +
 --
-.Example Kafka user configuration with ACL rules for producing and consuming from `my-topic`
+.Example Kafka user configuration with ACL rules to allow all operations on `my-topic` and `my-cluster`
```

Comment:
@ShubhamRwt I changed the ACLs set up in _Generating reassignment JSON files_ procedure so that it applies to the two scaling procedures.

Reply:
I asked Jakub regarding this. It not secure to use this ACL. I will try to find the more specific ones needed here. And probably we will not have to update this

Reply:
Okay. Thanks, Shubham. If we can show config that applies to reassign and works for teh scaling procedures too, that would be great.

Reply:
Sure, I am working over it.

Reply:
I have updated the doc with the specific ACL that we require and the previous documentation look fine as it is since there is no ACL change for the topics, just a new ACL for cluster is introduced

Reply:
I've reverted to the 3 x topic ACL rules + the updated cluster rule (`alter`)